dev-app/src/screens/CollectCardPaymentScreen.tsx

```diff
@@ -203,6 +215,13 @@ export default function CollectCardPaymentScreen() {
           requestedPriority: inputValues.requestedPriority,
         },
         captureMethod: inputValues?.captureMethod,
+        offlineBehavior: inputValues?.offlineBehavior,
+        offlineModeTransactionLimit: Number(
+          inputValues?.offlineModeTransactionLimit
+        ),
+        offlineModeStoredTransactionLimit: Number(
+          inputValues?.offlineModeStoredTransactionLimit
+        ),
```

Comment:
Per my previous comment, these should not be parameters to `createPaymentIntent`.